### ISSUE TITLE
List layers and maps across all users

### ISSUE THREAD

**Issue by raitisbe** [jirik/layman#17]
Could we have some option to make a composition public e.e visible to everyone at the same time allowing the author to modify or delete it?

I was thinking that maybe the author could call REST:
`/rest/<user>/maps/<mapname>/publish`

and then the same map would appear in two places
`GET /rest/<user>/maps/<mapname>`
and also
`GET /rest/public/maps/<mapname>`
and
`GET /rest/public/maps/`

**Comment by jirik** [comment]
Currently, there is neither authentication nor authorization in layman, so every map composition is visible to anyone and editable by anyone.

Authentication and authorization is the next major step in Layman's development. The discussion already started, detailed timeline should be available in August. With authentication and authorization, author will be able to set access rights of the map (e.g. visible to anyone, editable only by me).

Anyway, I expect REST API URLs remain same as now. Maps will be obtained simply by `GET /rest/<user>/maps` - and the response may contain only limited list of maps depending on user credentials and access rights.

**Comment by jirik** [comment]
Authentication and authorization are already implemented. Request all layers/maps will be possible after #257 is implemented.
